packages/emptystate/src/svg/document.svg

```diff
@@ -1,12 +1,12 @@
+    <path fill="currentColor" d="M38.0322,12.2366 C38.0322,11.8462 38.1564,11.6006 38.2979,11.4568 C38.438,11.3145 38.6718,11.1936 39.043,11.1936 L98.7634,11.1936 C99.135,11.1936 99.368,11.3145 99.508,11.4568 C99.65,11.6006 99.774,11.8462 99.774,12.2366 L99.774,92.7312 C99.774,93.3553 99.267,93.7742 98.7634,93.7742 L93.6451,93.7742 L93.6451,95.774 L98.7634,95.774 C100.27,95.774 101.774,94.5586 101.774,92.7312 L101.774,12.2366 C101.774,11.4011 101.496,10.6252 100.934,10.0539 C100.37,9.4812 99.599,9.1936 98.7634,9.1936 L39.043,9.1936 C38.2077,9.1936 37.4361,9.4812 36.8725,10.0539 C36.3103,10.6252 36.0322,11.4011 36.0322,12.2366 L36.0322,19.7742 L38.0322,19.7742 L38.0322,12.2366 Z"/>
+    <path stroke="currentColor" stroke-linecap="round" stroke-width="2" d="M59 41L80 41M37 51L80 51M38 61L80 61M38 71L80 71M38 81L80 81M38 91L63 91"/>
+    <circle cx="76.5" cy="92.5" r="4.5" stroke="currentColor" stroke-width="2"/>
+    <rect width="16" height="10" x="35" y="31" fill="currentColor" fill-opacity=".2" stroke="currentColor" stroke-width="2" rx="1"/>
+    <path fill="currentColor" d="M24,49.25 L7,49.25 C5.75736,49.25 4.75,48.2426 4.75,47 L4.75,21 C4.75,19.7574 5.75736,18.75 7,18.75 L35,18.75 L35,16.25 L7,16.25 C4.37665,16.25 2.25,18.3766 2.25,21 L2.25,47 C2.25,49.6234 4.37665,51.75 7,51.75 L24,51.75 L24,49.25 Z M24,89.25 L17.2903,89.25 C13.8571,89.25 11.25,92.4237 11.25,96.105 L11.25,112.895 C11.25,116.576 13.8571,119.75 17.2903,119.75 L47.7097,119.75 C51.1429,119.75 53.75,116.576 53.75,112.895 L53.75,109 L50.9274,109 L50.9274,112.895 C50.9274,115.114 49.3895,116.724 47.7097,116.724 L17.2903,116.724 C15.6105,116.724 14.0726,115.114 14.0726,112.895 L14.0726,96.105 C14.0726,93.8861 15.6105,92.2763 17.2903,92.2763 L24,92.2763 L24,90.7763 L17.2903,90.7763 C16.8256,90.7763 16.3811,90.85 15.9637,90.9864 C16.3875,90.8322 16.8339,90.75 17.2903,90.75 L24,90.75 L24,89.25 Z M103,81.75 L120.512,81.75 C123.321,81.75 125.75,79.7022 125.75,77 L125.75,53 C125.75,50.2978 123.321,48.25 120.512,48.25 L103,48.25 L103,50.75 L120.512,50.75 C122.04,50.75 123.128,51.8362 123.128,53 L123.128,77 C123.128,78.1638 122.04,79.25 120.512,79.25 L103,79.25 L103,81.75 Z M103,41.75 L115,41.75 C117.623,41.75 119.75,39.6234 119.75,37 L119.75,26 C119.75,23.3766 117.623,21.25 115,21.25 L103,21.25 L103,23.75 L115,23.75 C116.243,23.75 117.25,24.7574 117.25,26 L117.25,37 C117.25,38.2426 116.243,39.25 115,39.25 L103,39.25 L103,41.75 Z M47.7097,118.224 C48.1744,118.224 48.6189,118.15 49.0362,118.014 C48.6124,118.168 48.1661,118.25 47.7097,118.25 L17.2903,118.25 C16.8339,118.25 16.3875,118.168 15.9637,118.014 C16.3811,118.15 16.8256,118.224 17.2903,118.224 L47.7097,118.224 Z" opacity=".2"/>
+    <path fill="currentColor" d="M26,92.3226 L26,107.032 L89.7419,107.032 L89.7419,51.871 L89.7419,28.5806 L26,92.3226 Z M76.5,100 C80.6421,100 84,96.642 84,92.5 C84,88.3579 80.6421,85 76.5,85 C72.3579,85 69,88.3579 69,92.5 C69,96.642 72.3579,100 76.5,100 Z" opacity=".1"/>
```

Comment:
Not sure you meant to change this one? See svgr config https://github.com/pluralsight/design-system/blob/dc50ea0178ed2e8dca0aa82949c568ed5786380c/packages/emptystate/svgr.config.js#L33-L36